src/Components/Alert/Alert_Color_Options.php

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Alert\Components\Alert;
+
+class Alert_Color_Options implements Color_Options_Manager {
+
+	public const CSS_CLASS_PREFIX = 'tribe_alerts--';
```

Comment:
```suggestion
	public const CSS_CLASS_PREFIX = 'tribe-alerts-theme--';
```
Let's update this to match our existing classes, every class we use is prefixed with `tribe-alerts` and not `tribe_alerts`